nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -38,8 +38,11 @@ let
       cd "$HOME"
 
       # Allow the user to execute commands at the beginning of the X session.
+      # Prefer .xprofile over .profile as most people probably won't know about .xprofile
       if test -f ~/.xprofile; then
           source ~/.xprofile
+      elif test -f ~/.profile; then
+          source ~/.profile
```

Comment:
Sorry for being nitpicky still, but before merging this I'm not sure if it makes sense to prioritize `.xprofile` over `.profile`. 

They are completely unrelated and I think we'd still want to load `.profile` even if `.xprofile` exists. I can see why we might want to do this to avoid breakage, but `.xprofile` shouldn't be sourcing `.profile` anyway. Also anyone who's setup their machine to source `~/.profile` some other way, will also encounter breakage once this is merged (because `~/.profile` will then be sourced twice).

Since `/etc/profile` & `.profile` are bash specific, I think it would make sense to group them together.

Reply:
Yeah, I would've liked that we use that `Xsession` directly, but it doesn't state why the wrapper was introduced in the wrapper itself. Your nitpick is reasonable. I'll make the change :slightly_smiling_face:

Reply:
I think we use our own for consistency across DMs

Reply:
Alright, we should be good to go @kira-bruneau . I tested the changes and, as expected, `$HOME/.profile` is sourced and the exported env vars are there.